fund.py

```diff
@@ -0,0 +1,23 @@
+tuple_value = {"name": "nawal", "program": "python", "level": "beginer"}
+
+new_dict = {steps: value for (steps, value) in tuple_value.items() }
+update_value = new_dict.values()
+print(f"values {new_dict.values()}")
+
+new_tuple = {"org": "value"}
+for dict in new_dict:
+    tuple_value.update(new_tuple)  
+print(tuple_value)
```

Comment:
Create a separate file for each concept. I see two different concepts in this file. Also name the file which makes sense that way it would be easier for you to search in future.